pkg/registry/server/server.go

```diff
@@ -354,6 +354,28 @@ func ResolveAndMergeConfigsAndInjectTest(configs Getter, resolver Resolver, reso
 				image.Ref = ref
 				mergedConfig.Images = append(mergedConfig.Images, image)
 			}
+
+			// Attempt to handle a few simple raw_step types on a best-effort basis
+			for i := range config.RawSteps {
```

Comment:
The lines are quite long and they don't even fit my screen. It would be nice if you could break them apart like so:
```golang
to := fmt.Sprintf("%s-%s", rawStep.RPMImageInjectionStepConfiguration.To, ref)
modifiedStep.RPMImageInjectionStepConfiguration.To = api.PipelineImageStreamTagReference(to)
from := fmt.Sprintf("%s-%s", rawStep.RPMImageInjectionStepConfiguration.From, ref)
modifiedStep.RPMImageInjectionStepConfiguration.From = api.PipelineImageStreamTagReference(from)
```